R/module_transform_data.R

```diff
@@ -155,7 +155,14 @@ srv_transform_teal_data <- function(id, data, transformators, modules = NULL, is
             })
           })
 
-          data_out
+          # Ignoring unwanted reactivity breaks during initialization
+          data_return <- reactive({
+            if (is.null(data_out())) {
+              rlang::abort(class = "shiny.silent.error")
+            }
+            data_out()
+          })
+          data_return
```

Comment:
isn't `req()` enough?
```suggestion
          reactive({
            req(data_out())
          })
```